### ISSUE TITLE
All vorto data disappears after restart

### ISSUE THREAD

**Issue by hristobojilov** [eclipse/vorto#2595]
I'm using modified version of application-local.yml profile with my user data.
H2 in-memory db is not used and the data is saved in ${user.home}/.vorto/repo/binaries file.
Every time after I send SIGKILL to vorto repository process and run it again all my data disappears including my namespace and all imported models.
Is this config issue or a bug? How can I resolve it and preserve my data?

**Comment by kolotu** [comment]
Hi @hristobojilov this sounds like a config issue - I don't have that issue locally. 
Which value does the `spring.hibernate.ddl-auto` property have in your configuration?

**Comment by hristobojilov** [comment]
I googled that property and I found only about spring.jpa.hibernate.ddl-auto.
I tried to run it with **create**(to build DB schema first from scratch) and then switch to **validate**.  The result was that I'm getting an error 'Schema-validation: missing table [comment]'. I tried with **none** and turning on spring.jpa.generate-ddl with true/false different combinations with spring.jpa.hibernate.ddl-auto create/update/validate and I'm stuck that error above again. Either my data disappears or the app starts with an error.
